src/main/resources/vm/script/datamanagement/uploadFunctions.vm

```diff
@@ -182,6 +182,11 @@ function uploadGirderFile {
     local fileId=`echo $URI | sed -r 's/^.*[?&]fileid=([^&]*)(&.*)?$/\1/i'`
     local token=`echo $URI | sed -r 's/^.*[?&]token=([^&]*)(&.*)?$/\1/i'`
 
+    if [ ! `which girder-client` ]
+    then
+        pip install --user girder-client
```

Comment:
Also exit if it fails here.

Reply:
Added a test to exit if installation fails.